server/controllers/finance/creditorGroups.js

```diff
@@ -94,6 +114,26 @@ function update(req, res, next) {
     .done();
 }
 
+/**
+* DELETE /creditor_groups/:uuid
+*
+* Delete an existing creditor group
+*/
+function remove(req, res, next) {
+  const sql = 'DELETE FROM creditor_group WHERE uuid = ?;';
+  const uid = db.bid(req.params.uuid);
+  db.exec(sql, [uid])
+    .then(() => {
```

Comment:
You actually don't have to do two SQL queries.  Just check that the affectedRows is greater than 0.

For example:
```js
db.exec(sql, [uid])
  .then(result => {
    if (!result.affectedRows) {
      throw FAILURE_DELETING_CREDITOR_GROUP;
    }

    res.sendStatus(203);
  });
```